packages/headless/src/features/generated-answer/__snapshots__/generated-answer-analytics-actions.test.ts.snap

```diff
@@ -5,7 +5,7 @@ exports[`generated answer analytics actions > when analyticsMode is \`next\` > s
   "Rga.AnswerAction",
   {
     "action": "copyToClipboard",
-    "answerId": "456",
+    "answerId": "123",
```

Comment:
If we want to be a tad bit more accurate, this has to be a UUID.
Don't know if there's much value in making it real-er, but FYI.

Here's one if you want :D `94b77748-2479-4e4b-a4e8-010fa62b04a0`